site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Getting the NGINX Gateway Fabric Image with JWT
+description: "Use your NGINX Gateway Fabric subscription JWT token to get the NGINX Plus Gateway Fabric image from the F5 Docker registry."
+weight: 150
+doctypes: [""]
+toc: true
+---
+
+{{<custom-styles>}}
+
+## Overview
+
+Follow the steps in this document to pull the NGINX Plus Gateway Fabric image from the F5 Docker registry into your Kubernetes cluster using your JWT token.
+
+{{<important>}}
+An NGINX Plus subscription certificate and key will not work with the F5 Docker registry.
+
+For NGINX Gateway Fabric, you must have an NGINX Gateway Fabric subscription -- download the NGINX Plus Gateway Fabric (per instance) JWT access token from [MyF5](https://my.f5.com).
```

Comment:
I'm assuming the subscription will be named something else. PM can probably help with that. ( I think it'll be Connectivity Stack for Kubernetes)

Reply:
Yes, will most definitely be Connectivity Stack for kubernetes but i will confirm with Mike